src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.html

```diff
@@ -1,8 +1,8 @@
-<action-button [command]="addContainer()"></action-button>
+<dot-action-button [command]="addBox()"></dot-action-button>
```

Comment:
Every time you use an old component that doesn't have the ```dot-``` prefix please add it, let's try to make our source code as great as we can one step at a time.